testsuite/features/secondary/srv_cobbler_sync.feature

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2022 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Run Cobbler Sync via WebUI
+
+  Scenario: Login as admin
+    Given I am authorized for the "Admin" section
+
+  Scenario: Sanity check for page existance
```

Comment:
Please be more precice here. This Scenario description is to generic.

Reply:
Right, I have changed it comitting now